courses/free-courses-es.md

```diff
@@ -59,6 +60,11 @@
 * [Principios de SQL](https://programadorwebvalencia.com/cursos/sql/introducci%C3%B3n/) - Andros Fenollosa (Programador Web Valencia)
 
 
+#### NoSQL
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. can just use Bases de Datos